rubi/pyproject.toml

```diff
@@ -1,27 +1,28 @@
 [tool.poetry]
 name = "rubi"
-version = "1.3.2"
-description = "a python SDK for the Rubicon Protocol"
-authors = ["denver <denver@rubicon.finance>"]
+version = "2.0.0"
+description = "A python SDK for the Rubicon Protocol"
+authors = ["denver <denver@rubicon.finance>", "adam <adam@rubicon.finance>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.10"
```

Comment:
we will probably want to add some type of bot to help with dependencies on the repo side, like [this](https://github.com/dependabot). think it makes the most sense to keep python version as variable, will just want to keep an eye on this whenever new updates come out 🫡